styles.css

```diff
@@ -133,19 +135,55 @@ div.project-item {
 	margin: 5px;
 }
 
+div.project-item p {
+	margin-right: 20%;
+}
+
 button.live, .code {
```

Comment:
Possibly make it so the buttons display horizontally instead of vertically - especially helps prevent mobile users from accidentally tapping the wrong button and can look a bit tidier

tried getting it to work by adjusting flex-direction but couldn't, so I'm not exactly sure how to go about it

Reply:
Tried that, just given up for now... As the buttons are centered, I'd say it's alright for now.